src/eval.c

```diff
@@ -247,7 +247,7 @@ void scriptingInit(int setup) {
                                 "    i = dbg.getinfo(3,'nSl')\n"
                                 "  end\n"
                                 "  if i then\n"
-                                "    return i.source .. ':' .. i.currentline .. ': ' .. err\n"
+                                "    return err .. '. ' .. i.source .. ':' .. i.currentline\n"
```

Comment:
Maybe this is a bit nicer:
```suggestion
                                "    return err .. ' (' .. i.source .. ':' .. i.currentline .. ')'\n"
```